index.bs

```diff
@@ -257,28 +264,112 @@ spec:css-syntax-3;
+    3.  If |document| has no [=document/browsing context=], return `false`.
 
-  3.  If |document| has no [=document/browsing context=], return `false`.
+    4.  Let |origin| be |settings|' [=environment settings object/origin=].
 
-  4.  Let |origin| be |settings|' [=environment settings object/origin=].
+    5.  Let |current| be |document|'s [=document/browsing context=].
 
-  5.  Let |current| be |document|'s [=document/browsing context=].
+    6.  While |current| has a [=parent browsing context=]:
+
+        1.  Set |current| to |current|'s [=parent browsing context=].
+
+        2.  If |current|'s [=active document=]'s [=origin=] is not [=same origin=] with |origin|,
+            return `false`.
+
+    7.  Return `true`.
+  </ol>
+
+  ### Registry: Credential Type to Options Member Identifier Mappings ### {#sctn-registry-cred-type-to-options-member-identifier}
+
+  This registry maps [=credential types=] (i.e., {{Credential/[[type]]}} values) to the
+  [=dictionary member=] [=identifier=] used in
+  {{CredentialCreationOptions}} and {{CredentialRequestOptions}} (i.e., "options dictionaries")
+  by their specifications.
+
+  Note: This registry is used by the [=relevant credential interface objects=] algorithm.
+
+
+  <table class="complex data longlastcol">
+    <tr>
+        <th>Credential Type<br>
+        (in alphabetical order)</th>
+        <th>Options Member Identifier</th>
+        <th>Specification</th>
+        <th>Requestor Contact</th>
+    </tr>
+    <tr>
+        <td>federated</td>
+        <td>federated</td>
+        <td>This specification: [[#federated]]</td>
+        <td><a href="https://www.w3.org/2011/webappsec/">W3C</a></td>
+    </tr>
+    <tr>
+        <td>otp</td>
+        <td>otp</td>
+        <td>[[WEB-OTP]]</td>
+        <td><a href="https://wicg.io/">WICG</a></td>
+    </tr>
+    <tr>
+        <td>password</td>
+        <td>password</td>
+        <td>This specification: [[#passwords]]</td>
+        <td><a href="https://www.w3.org/2011/webappsec/">W3C</a></td>
+    </tr>
+    <tr>
+        <td>public-key</td>
+        <td>publicKey</td>
+        <td>[[WEBAUTHN]]</td>
+        <td><a href="https://www.w3.org/blog/webauthn/">W3C</a></td>
+    </tr>
+  </table>
+
+  <!-- NOTE: The below Registration Entry Requirements and Update Process is based on
+  https://www.w3.org/TR/timing-entrytypes-registry/, and is similar to other W3C
+  "registry" specs. I.e., this Requirements and Update Process is typical for
+  W3C registries. -->
+
+  #### Registration Entry Requirements and Update Process #### {#sctn-registry-requirements}
+
+  * Each [=credential type=] must be unique amongst the set of [=credential types=].
+
+  * Each registry entry must state the [=dictionary member=] [=identifier=] used in the
+    credential specification's extentions of {{CredentialCreationOptions}} and
+    {{CredentialRequestOptions}}.
+
+  * Each registry entry must include a link that references a publicly available specification (PAS)
```

Comment:
No need to introduce an abbreviation that isn't used elsewhere:
```suggestion
  * Each registry entry must include a link that references a publicly available specification
```

Reply:
I suppose. FWIW, It is in there because the existing formal registries I cut & pasted that text from had that in it.